docs/okeps/okep-5259-no-overlay.md

```diff
@@ -0,0 +1,1491 @@
+create networks (default or CUDN), with no-overlay mode enabled, allowing
+traffic to skip Geneve encapsulation (i.e., the overlay network) and simply make
+use of the learnt routes in the underlay or provider network for inter-node
+communication.
+
+While BGP is the proposed solution for exchanging east-west routing information
+within a cluster, it is not the only possible option. Future implementations of
+no-overlay mode may incorporate additional routing technologies and protocols
+beyond BGP.
+
+## User-Stories/Use-Cases
+
+### Story 1: Have the cluster default network in the no-overlay mode
+
+As a cluster admin, I want to enable no-overlay mode for the cluster default
+network to integrate seamlessly with existing BGP-capable networks, achieve
+maximum network performance, and reduce overhead.
+
+### Story 2: Create a CUDN in the no-overlay mode
+
+As a cluster admin, I want to enable no-overlay mode for a cluster user defined
+network to integrate seamlessly with existing BGP-capable networks and achieve
+maximum network performance.
+
+### Story 3: Deploy networks in the no-overlay mode without external BGP routers
+
+As a cluster admin, I want to use no-overlay mode for intra-cluster traffic
+without advertising pod networks to the external network or depending on
+external BGP routers.
+
+## Proposed Solution
+
+This solution leverages the existing BGP feature to advertise each node's pod
+subnet via the CNCF project [FRR-K8s](https://github.com/metallb/frr-k8s). FRR
+routers managed by FRR-k8s then propagate these routes throughout the provider
+network, enabling direct pod-to-pod traffic without an overlay. A single cluster
+can simultaneously have networks in both overlay and no-overlay modes.
+
+When no-overlay mode is enabled for a network:
+
+* By default, when BGP advertisement is enabled, the BGP feature disables SNAT
+  for pod egress traffic leaving nodes. For users who only want to have
+  no-overlay mode but do not want to expose pod IPs externally, this behavior is
+  not ideal. To address this, users will be able to enable SNAT for
+  pod-to-external traffic. This will be configured via a new `outboundSNAT`
+  field in the `ClusterUserDefinedNetwork` CRD for CUDNs, and the
+  `no-overlay-outbound-snat` flag for the default network.
+
+* For east-west traffic:
+  * Intra-node traffic remains unchanged, handled by the local OVN bridge.
+  * Inter-node traffic will follow the same path as north-south traffic, routed
+    via the underlay. Pod-to-pod, Pod-to-clusterIP traffic within the same
+    network will be forwarded by the provider network without encapsulation or
+    NAT. Pod-to-node egress traffic will be SNATed to the node IP at the source
+    node to allow a pod to access nodePort services in a different UDN.
+
+### Configuration File and Flags
+
+For the default network transport, the configuration file will have a new section
+`[default-network]`, with the following new flags:
```

Comment:
There is already a section called `[default]` which contains, among other things, the MTU for the default network. Isn't that existing section appropriate to hold these parameters?